src/buildVersions/BuildVersions.ts

```diff
@@ -46,7 +46,7 @@ export class BuildVersions {
     return dataAccessor.dataAsync()
   }
 
-  public async saveAsync(message = 'update build versions'): Promise<boolean> {
+  public async saveAsync(message : string): Promise<boolean> {
```

Comment:
see note below; this should be a optional param `message?: string` and a default should be set on the first line such as

```
message = message ?? 'update build versions'
```

unfortunately, I believe the [??=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) operator is not available in the version of typescript this project currently uses :frowning: